kzz7yh-a54197/cod-ve07v1_kzz7yh-a54197.xml

```diff
@@ -377,8 +377,8 @@
           diuer<lb ed="#V" n="68" break="no"/>sitatis: que ibi incipit. Nunc inspiciamus: et diuiditur in partes 
           <lb ed="#V" n="69"/>duas: quia primo facit quod dictum est. Secundo distinguit 
           tripli<lb ed="#V" n="70" break="no"/>cem acceptionem nominis persona ibi Quod autem secundum substantiam etc. 
-          <lb ed="#V" n="71"/>Circa hanc Di. quaeruntur duo. Po de persone quiddi 
-          <lb ed="#V" n="72"/>tate. 2o de persone communitate. 
+          <lb ed="#V" n="71"/>Circa hanc Di. quaeruntur duo. Primo de persone quod 
+          diuini<lb ed="#V" n="72"/>tate. 2o de persone communitate. 
```

Comment:
"quidditate" is actually correct here. (It's a medievalism that refers to the "what-ness" of a thing.) So here the question is about the "what-ness of the person"

```suggestion
          <lb ed="#V" n="71"/>Circa hanc Di. quaeruntur duo. Primo de persone 
          quiddi<lb ed="#V" n="72" break="no"/>tate. 2o de persone communitate. 
```